### PR TITLE
Chore/iac/ansible/playbook coments

### PR DESCRIPTION
**Pull request by alecio-rocha**
check my PR!

### CODE REVIEW

File: iac/ansible/Requirements_SOAK [hunk 1]
```diff
@@ -0,0 +1,78 @@
+            Requirements
```
Comment (marcpires) [inline iac/ansible/Requirements_SOAK:1]: This file must be removed

**Review by marcpires — APPROVED** [review]
LGTM

Must merge onder PR first

File: iac/ansible/playbook.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      remote_src: yes
+    when: go.stat.exists == False
+
+  - name: Update system path
+    ansible.builtin.lineinfile:
+     path: /etc/environment
+     line: PATH="$PATH:/bin:/usr/bin:/usr/local/go/bin" # Add this line in file "environment"
+     backup: yes
+
+  - name: Get environment var values
```
[the remark this answers is not in the document]
Comment (alecio-rocha) [reply iac/ansible/playbook.yml:34]: Added task comment

File: iac/ansible/playbook.yml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    # Why not command instead of shell ?
+    # 1. operations like > | ; & does not work
+    # 2. Variables like $HOSTNAME will not work
+    ansible.builtin.shell: '. /etc/environment && echo $PATH'
+    register: envs_vars
+
+  - name: Get env vars values
```
[the remark this answers is not in the document]
Comment (alecio-rocha) [reply iac/ansible/playbook.yml:41]: Added task comment

File: iac/ansible/playbook.yml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    vars:
+      packages:
+      - docker
+      - docker.io
+      - containerd
+      - runc
+    when: ansible_lsb.id | lower == "ubuntu" or ansible_lsb.id | lower == "debian" # verify what version Linux for execute install.
+    register: docker_packages
+
+  - name: Setup repository packages
```
[the remark this answers is not in the document]
Comment (alecio-rocha) [reply iac/ansible/playbook.yml:59]: Added task comment

File: iac/ansible/playbook.yml [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+      - curl
+      - gnupg
+      - lsb-release
+      - local-apt-repository
+    when: ansible_lsb.id | lower == "ubuntu" or ansible_lsb.id | lower == "debian" # verify what version Linux for execute install.
+    register: repo_packages
+
+  - name: Add Docker GPG key
+    ansible.builtin.apt_key:
+      url: https://download.docker.com/linux/{{ ansible_lsb.id | lower }}/gpg # Downloading the file key, but is necessary to declare the version Linux because local of that file is different according version.
```
[the remark this answers is not in the document]
Comment (alecio-rocha) [reply iac/ansible/playbook.yml:76]: This explanation atualized

File: iac/ansible/playbook.yml [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+      id: 7EA0A9C3F273FCD8
+      keyring: /usr/share/keyrings/docker-archive-keyring.gpg # adding the key in the file
+      state: present
+    when: ansible_lsb.id | lower == "ubuntu" or ansible_lsb.id | lower == "debian"
+    register: gpg_repo
+
+  - name: Add Docker Repository
+    ansible.builtin.apt_repository:
+      # Get the key of the file so that you have access to the repository, but is necessary to declare the version Linux and  because local of that file is different according version.
```
[the remark this answers is not in the document]
Comment (alecio-rocha) [reply iac/ansible/playbook.yml:85]: This explanation has been updated

File: iac/ansible/playbook.yml [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+     name: "{{ packages }}"
+     state: present
+    vars:
+      packages:
+        - docker-ce 
+        - docker-ce-cli 
+        - containerd.io
+    register: docker_install
+
+  - name: Debug
```
[the remark this answers is not in the document]
Comment (alecio-rocha) [reply iac/ansible/playbook.yml:107]: Added task comment

File: iac/ansible/Requirements_SOAK (hunk 1 above)
Comment (alecio-rocha) [reply iac/ansible/Requirements_SOAK:1]: The file is removed

**Pull request closed without merging**